### ISSUE TITLE
bad value -1.000000 of time stats

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#798]
**Reported by saldsl on 4 Aug 39027542 18:13 UTC**
After installing Boinc 6.5.0 I get these error messages on startup:

```
ven 09 gen 2009 17:21:35 CET||[bad value -1.000000 of time stats connected_frac; ignoring
ven 09 gen 2009 17:21:35 CET||[error](error]) bad value -1.000000 of time stats active_frac; ignoring
```

I think this causes a very low value in "% of time BOINC client is running" as I see under my projects details.

**Comment by romw** [comment]
**Commented by davea on 10 Mar 39027590 00:26 UTC**
(In [16859]) - client: connected_frac -1 is OK; fixes #821

**Comment by romw** [comment]
**Commented by saldsl on 23 Feb 39029860 16:00 UTC**
What about the active_frac?

Every time I restart the boinc client and I issue an update for a project, I see that "% of time BOINC client is running" in computer stats drops to zero...

**Comment by romw** [comment]
**Commented by saldsl on 10 Oct 39029898 11:06 UTC**
I activated debug messages for time stats. This is what I get with 6.5.0:

```
sab 10 gen 2009 16:42:30 CET||[dt 210.942925 w2 0.999756 on 0.000000; active 1.000000; conn 1.000000
sab 10 gen 2009 16:42:40 CET||[time_debug](time_debug]) dt 10.001069 w2 0.999988 on 0.000012; active 1.000000; conn -1.000000
sab 10 gen 2009 16:42:50 CET||[dt 10.004007 w2 0.999988 on 0.000023; active 1.000000; conn -1.000000
sab 10 gen 2009 16:43:00 CET||[time_debug](time_debug]) dt 10.005994 w2 0.999988 on 0.000035; active 1.000000; conn -1.000000
sab 10 gen 2009 16:43:10 CET||[dt 10.006003 w2 0.999988 on 0.000046; active 1.000000; conn -1.000000
```


And this is what I get with the old 6.4.5:

```
sab 10 gen 2009 16:45:22 CET||[time_debug](time_debug]) dt 189.808737 w2 0.999780 on 0.999780; active 1.000000; conn -1.000000
sab 10 gen 2009 16:45:32 CET||[dt 10.028259 w2 0.999988 on 0.999780; active 1.000000; conn -1.000000
sab 10 gen 2009 16:45:42 CET||[time_debug](time_debug]) dt 10.012002 w2 0.999988 on 0.999780; active 1.000000; conn -1.000000
```

As you can see, "on" in 6.5 starts from zero every time I start the client, in 6.4.5 it remembers previous stats. I think this bug should be reopened, correct me if I'm wrong.

**Comment by romw** [comment]
**Commented by saldsl on 13 May 39065725 23:33 UTC**
Seems it's resolved in 6.6.2
You can re-close this. Thanks.
